XU-TechnicalPaper.md

```diff
@@ -102,17 +102,60 @@ Payment channel atomic swap technology is fairly new and was just recently succe
+2. The Taker, the impatient buyer or seller filling above limit orders. Usually in the form of a market order.
+
+A description of how we realize above goals:
+
+1. Minimize latency of order updates
+For orders, the DOB protocol strictly follows the first come, first served principle, which remains fully compatible with how order book systems of exchanges work. To get the best achievable latency between two nodes which intend to receive order updates from each other, we require an open connection between two nodes on Internet Protocol level. From a XU network topology point of view we require a direct connection without intermediary hop with all nodes a node wants to receive order information from. Transport Layer design choices using socket connections and [efficient congestion control mechanisms](https://github.com/google/bbr) are currently in the works.
+
+2. Single point of execution 
+There is one single point of execution for each order: the maker. The maker decides on which incoming order gets to fill the order. All orders contain payment information (e.g. a lightning invoice). Lightning is defined in a way (See (BOLT #11))[https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md], that an invoice can only be paid by the first one to successfully send a payment to an invoice issuer, in our case the maker. Also lightning takes care that a taker has a payment channel path with sufficient volume for a specific order. If the incoming order can't be filled because it was already filled by someone else or canceled, the makers responsibility is to send the funds back in a timely manner, otherwise there will be a punishment using its staked XUC (details tbd). Worst-case the takers payment is locked for the the time as specified in the Atomic Swap HTLC and the maker gets punished. Under no circumstances a permanent loss of funds can occur.
+
+Sidenote: Transferring order execution authority to a third party, in particular to a decentralized consensus, was considered and deemed not feasible for the following reasons:
+a) consensus=slow
+b) hard to sell; an exchange won't let anyone else decide on who fills its users orders
+c) hard to control since in our case exchanges could always treat their local order book with priority
+
+3. Reward liquidity providers
+Makers are earning XUC, takers are paying in XUC for filling orders. Makers set a price for each order in a specified fee field. We aim to establish a fee market where makers are competing with a combination of order quantity, price and XUC fee. A taker pays the full XUC fee via a default Raiden Channel to the maker. It's down to exchanges to take over the role of handling XUC payments and earnings for their users or integrate this as a business model for traders. 
+
+4. Punish malicious behavior
+Market makers are required to stake a certain amount of XUC in a smart contract in order to be able to act as a market maker. This will be used to execute punishments.
```

Comment:
This is a complicated topic and probably out of scope of these revisions, but I'm not sure how punishments are going to work here.

As I see it, coming to a consensus over which nodes should be punished may very well be impossible. However, individual nodes can ban misbehaving nodes according to their own discretion. To prevent sybil attacks - where banned nodes re-enter the network with new identifiers and/or IP addresses - I'd propose that each node that wants to participate in the network must register their public key using XUC. I'd actually prefer a burn of XUC such that misbehaving and being banned incurs a permanent cost, but if we go with the staking route I think the stake should be locked for a substantial amount of time to truly discourage and prevent spam/sybil attacks.

Reply:
I'll add that in a future commit.